packages/database/src/migrations/20220822035354-AddSyncStatusToSyncGroup-modifies-schema.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  return db.runSql(
+    `ALTER TABLE data_service_sync_group ADD COLUMN sync_status text DEFAULT 'IDLE'`,
```

Comment:
Can we restrict the value to only be IDLE, SYNCING, ERROR?

Reply:
Fair point. I was thinking of this, but then remembered @mcccclean had mentioned something in dev-chat about pain points when working with enum types in postgres. I recall a time when you had to fully delete them and re-create the type if you wanted to add a value, but it seems postgres has supported adding types ad hoc since version 9...

I agree, enum type is nicer here, I'll do it 👍